ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/service/FragmentationConfigCreationFunction.java

```diff
@@ -15,7 +14,6 @@
 
 import static be.vlaanderen.informatievlaanderen.ldes.server.domain.constants.RdfConstants.FRAGMENTATION_TYPE;
 import static be.vlaanderen.informatievlaanderen.ldes.server.domain.constants.RdfConstants.RDF_SYNTAX_TYPE;
-import static be.vlaanderen.informatievlaanderen.ldes.server.domain.view.service.ViewSpecificationConverter.FRAGMENTATION_NAME;
 
 public class FragmentationConfigCreationFunction implements Function<RDFNode, FragmentationConfig> {
 	@Override
```

Comment:
This method has become quite long and complicated, maybe readability could be improved by delegating some logic to smaller private methods?